edgelet/external-provisioning/src/apis/external_provisioning_api.rs

```diff
@@ -110,4 +115,59 @@ where
                 }),
         )
     }
+
+    fn reprovision_device(
+        &self,
+        api_version: &str,
+    ) -> Box<dyn Future<Item = (), Error = Error<serde_json::Value>> + Send> {
+        let configuration: &configuration::Configuration<C> = self.configuration.borrow();
+
+        let method = hyper::Method::GET;
+
+        let query = ::url::form_urlencoded::Serializer::new(String::new())
+            .append_pair("api-version", &api_version.to_string())
+            .finish();
+        let uri_str = format!("/device/reprovision?{}", query);
+
+        let uri = (configuration.uri_composer)(&configuration.base_path, &uri_str);
+        // TODO(farcaller): handle error
+        // if let Err(e) = uri {
+        //     return Box::new(futures::future::err(e));
+        // }
+        let mut req = hyper::Request::builder();
+        req.method(method).uri(uri.unwrap());
+        if let Some(ref user_agent) = configuration.user_agent {
+            req.header(http::header::USER_AGENT, &**user_agent);
+        }
+
+        let req = req
+            .body(hyper::Body::empty())
+            .expect("could not build hyper::Request");
+
+        //        if let Some(ref sas_token) = configuration.sas_token {
```

Comment:
Delete this commented-out code.

(And also delete it from `get_device_provisioning_information`)